### PR TITLE
TEAMNADO-3874: Bug: Blank view does not show appropriate message

### PR DESCRIPTION
**Pull request by cowanrc**
Adds "No Results Found" when Filters have 0 subscriptions

### CODE REVIEW

File: src/components/ProductsTable/ProductsTable.tsx [hunk 1]
```diff
@@ -251,7 +254,7 @@ const ProductsTable: FunctionComponent<ProductsTableProps> = ({
           ))}
         </Tbody>
       </TableComposable>
-      {paginatedProducts.length == 0 && data.length > 0 && (
+      {paginatedProducts.length == 0 && data.length >= 0 && (
```
Comment (jlgre) [inline src/components/ProductsTable/ProductsTable.tsx:257]: I'm confused about the logic of this if check. Shouldn't it show no results if either of these values are strictly equal to 0? Also is there ever a case where paginatedProducts.length is not 0 when data.length is 0? I feel like it works if we just check len of paginated products

Comment (cowanrc) [reply src/components/ProductsTable/ProductsTable.tsx:257]: I've removed all logic minus the paginatedProducts.length logic, and it seems to work the same. I can push with just that logic

Comment (jlgre) [reply src/components/ProductsTable/ProductsTable.tsx:257]: Cool, yeah if the behavior is the same I think that's a cleaner and more correct implementation

**Review by jlgre — APPROVED** [review]
Looks good

**Pull request merged**